sos/plugins/openvswitch.py

```diff
@@ -90,6 +124,13 @@ def setup(self):
             "ovs-appctl dpctl/show -s",
             # Capture DPDK datapath flows
             "ovs-appctl dpctl/dump-flows",
+            # The '-s' option enables dumping of packet counters on the
+            # ports.
+            "ovs-dpctl -s show",
+            # Capture the in-kernel flow information if it exists
+            "ovs-dpctl dump-flows -m",
+            # Capture the flow information also for offloaded rules
+            "ovs-dpctl dump-flows type=offloaded -m",
```

Comment:
I cant find them in upstream anywhere - why do you need them in legacy branch but not in upstream?